com.unity.render-pipelines.universal/ShaderLibrary/Input.hlsl

```diff
@@ -33,15 +33,9 @@ struct InputData
     half3   bakedGI;
     float2  normalizedScreenSpaceUV;
     half4   shadowMask;
-
-    #if defined(LIGHTMAP_ON)
     half2   lightmapUV;
-    #else
     float3  vertexSH;
-    #endif
-    #if defined(_NORMALMAP)
     half3x3 tangentToWorld;
-    #endif
```

Comment:
Is this change intentional?

Reply:
Yes, @eh-unity and @lukaschod thought that since this is just an intermediate struct, not varying struct, the convention has been to leave out the ifdefs from the struct definition and let compiler optimize out unused stuff.

Reply:
And just to clarify, these 3 fields that the ifdefs are protecting were all added in the first debug views PR: https://github.com/Unity-Technologies/Graphics/pull/1799/files#diff-cb824df074d9e71d1b765eee08de28a09813837fc84b3aef29c0b7184dcacabe

Reply:
Great! After some discussion and testing the ifdef checks are unnecessary as the variables will be stripped if not used.